packages/nimble-components/src/utilities/unit-format/unit-scale/tests/byte-unit-scale.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { parameterizeNamedList } from '../../../tests/parameterized';
+import { byteUnitScale } from '../byte-unit-scale';
+
+describe('ByteUnitScale', () => {
```

Comment:
I think we should have tests in at least one more language here. We don't need all languages since we're relying on NumberFormatter for translations but we should test one non-English language to ensure we don't hardcode an `en` somewhere in the implementation.